pallets/delegation/src/lib.rs

```diff
@@ -132,10 +132,21 @@ pub mod pallet {
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 		#[pallet::constant]
 		type MaxSignatureByteLength: Get<u16>;
+
+		/// Maximum number of revocations.
 		#[pallet::constant]
 		type MaxRevocations: Get<u32>;
+
+		/// Maximum number of upwards traversals of the delegation tree from a
+		/// node to the root and thus the depth of the delegation tree.
 		#[pallet::constant]
 		type MaxParentChecks: Get<u32>;
+
+		/// Maximum number of all children for a delegation node. For a binary
+		/// tree, this should be twice the maximum depth of the tree, i.e.
+		/// `2 ^ MaxParentChecks`.
+		#[pallet::constant]
+		type MaxChildren: Get<u32> + Clone;
```

Comment:
Because the compiler otherwise complains about `DelegationNode` not being clonable because `MaxChildren` does not implement `Clone`. I feel like I must have done something wrong...

Reply:
https://stackoverflow.com/a/39415236/12913324

Reply:
Looks like it's this issue

Reply:
I suppose we don't want to introduce another dependency [(rust-derivative)](https://github.com/mcarton/rust-derivative) to solve this as recommended in the open rust issue?

Reply:
yeah i don't think so. I'm fine with the way it currently is.